packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -26,25 +26,22 @@ export const UnitRowRecoveryKey = () => {
   const { recoveryKey } = useAccount();
   const alertBar = useAlertBar();
   const [modalRevealed, revealModal, hideModal] = useBooleanState();
-  const [errorText, setErrorText] = useState<string>();
-  const onError = (e: Error) => {
-    setErrorText(e.message);
-    hideModal();
-    alertBar.show();
-  };
 
-  const [getAccount, { accountLoading }] = useLazyAccount((error) => {
-    setErrorText('Sorry, there was a problem refreshing the recovery key.');
-    alertBar.show();
+  const [getAccount, { accountLoading }] = useLazyAccount(() => {
+    hideModal();
+    alertBar.error('Sorry, there was a problem refreshing the recovery key.');
```

Comment:
Do we need to send in `error` to the alert bar here?

Reply:
Yes